source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -0,0 +1,136 @@
+---
+title: The Ember Times - Issue No. 55
+author: Kenneth Larsen, Amy Lam, Miguel Braga Gomes, Ryan Mark, Sivakumar Kailasam, Jessica Jordan, Alon Bukai
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/13-the-ember-times-issue-55.html"
+responsive: true
+---
+
+nuqneH Emberistas! 🐹
+
+Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/13/the-ember-times-issue-55.html) or in our [e-mail newsletter](https://the-emberjs-times.ongoodbits.com/2018/07/13/the-ember-times-issue-55) what has been going on in Emberland this week.
+
+This week you can learn about **updating** your ember-cli app 🖳. Learn from firsthand experience how to become an **addon maintainer** 💪. Get some tips on writing your own **RFCs** ✨. Check out the new **SEO tactics** of Ember 🔎 and last but not least learn about the potential move of Ember to a new **chat platform** 💬. Go ahead and enjoy!
+
+---
+
+## [Don't Worry, Ember CLI Got You Covered](https://github.com/ember-cli/ember-cli-update) 🖳
+The **number one** tool for updating Ember.js apps or addons just got **even better**. The [newest version](https://github.com/ember-cli/ember-cli-update/releases) of `ember-cli-update` now runs `qunit-dom-codemod` for you. This means that you, with close to no effort at all, can utilize this great addon for your tests.
+
+And the cool things is that `ember-cli-update` fetches new codemods **during runtime** - so no need to update to get this nice codemod! To run the codemod just type `ember-cli-update --run-codemods` and magic will take care of the rest for you.
+
+You can visit the addon page for [ember-cli-update](https://github.com/ember-cli/ember-cli-update) to read all about it or visit [qunit-dom](https://github.com/simplabs/qunit-dom) to learn about all the nice features it provides.
+
+---
+
+## [Start Small - How I Became an Ember Addon Maintainer by Ryan Mark](github.com/tylerturdenpants) 💪
```

Comment:
This link currently is interpreted as an relative url on the review app:

https://ember-website-staging-pr-3452.herokuapp.com/blog/2018/07/13/github.com/tylerturdenpants

 If we want to make sure it leads away from the blog, I think this should be

```
https://github.com/tylerturdenpants
```